src/includes/getting-started-install/capacitor.mdx

```diff
@@ -0,0 +1,27 @@
+To install the Capacitor SDK, use npm or yarn:
+
+```bash {tabTitle:npm}
+npm install --save @sentry/capacitor @sentry/tracing
+```
+
+```bash {tabTitle:Yarn}
+yarn add @sentry/capacitor @sentry/tracing
+```
+
+Add the plugin declaration to your `MainActivity.java` file
+
+```groovy {filename:MainActivity.java}
+import io.sentry.capacitor.SentryCapacitor;
+
+public class MainActivity extends BridgeActivity {
+  @Override
+  public void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+
+    // Initializes the Bridge
+    this.init(savedInstanceState, new ArrayList<Class<? extends Plugin>>() {{
+      add(SentryCapacitor.class);
+    }});
+  }
+}
+```
```

Comment:
does the user need to do this manually? the other SDKs require only the `getting-started-config` config to be done, and the JS layer would init. the native SDKs automatically.

Reply:
Yes, this is typical for all Capacitor android configurations. I'd be happy to work on a solution for the SDK to automatically add this in. @bruno-garcia do you think we should update this?

Reply:
mm it's ok if we decided to not do it now, I just wanted to be sure that it's possible.

eg https://github.com/ionic-team/capacitor-plugins/tree/main/device#example
the example only touches the JS layer and everything else does it automatically, wondering if we should do the same with our Native plugin.

Reply:
React Native relies on our wizard: https://github.com/getsentry/sentry-wizard

For now lets add the manual step to unblock us. We can iterate on this

Reply:
`sentry-wizard` does some other stuff, it's not really setting up the Android plugin, but yeah lets get it unblocked and iterate.